storagegen/tmpl/minio1.tmpl

```diff
@@ -155,7 +155,11 @@ func (db *{{.TypeName}}MinioDb) initBucket(mc *minio.Client, useVersioning bool)
 }
 
 func (db *{{.TypeName}}MinioDb) filename(partnerID string, id string) string {
```

Comment:
We also need to remove partnerID here and from the other functions (Get, Create, Delete)

Reply:
tested on partner-service in the new wat that we discussed and it seems to work good.